superset-frontend/src/SqlLab/components/QueryTable/index.tsx

```diff
@@ -35,10 +35,12 @@ import ResultSet from '../ResultSet';
 import HighlightedSql from '../HighlightedSql';
 import { StaticPosition, verticalAlign, StyledTooltip } from './styles';
 
-interface QueryTableQuery extends Omit<Query, 'state' | 'sql' | 'progress'> {
+interface QueryTableQuery
+  extends Omit<Query, 'state' | 'sql' | 'progress' | 'output' | 'results'> {
```

Comment:
Any reason you need to omit `output` from `Query`?
Let's keep it if we don't have specific reason.

Reply:
because the output is no longer used, so it shouldn't be part of the interface.

Reply:
If you see the `Query`, there are a lot of other keys which we are not using for the table showing. 
Do we need to omit all of them?

Let's keep `output` if we don't need to change the type of `output`.